.github/workflows/ci.yml

```
@@ -0,0 +1,141 @@
+  ci-check-frontend:
+    name: ci-check-frontend
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./echo/frontend/
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
```

actionlint (1.7.4)

the runner of "actions/setup-node@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)